packages/coq/coq.8.13.2/opam

```diff
@@ -30,6 +30,9 @@ depends: [
 conflicts: [
   "ocaml-option-nnp"
 ]
+build-env: [
+  [COQ_USE_DUNE="false"]
```

Comment:
Actually, I'm not sure this will work; the test is coq's makefile is `ifndef COQ_USE_DUNE` , thus, I think the right fix is  unset COQ_USE_DUNE` before the `make world` call in the script itself.

Reply:
Currently it indeed doesn't work (as you can see in the CI). Frustratingly, I can't get anything to work locally at all, meaning that even when I checkout `master`, `cd`ing to, say, `packages/coq/coq.8.13.2` and doing `opam install .` fails (with `bwrap: execvp ./configure: No such file or directory`). However, the env log does show that `COQ_USE_DUNE` is set to `false` in this branch, which I take as a minor win.

Am I just testing things wrong?

Reply:
I think the way to test things is indeed a bit different; I'm not an expert, but the way I test this kind of stuff, is to do `opam add local $path_to_my_repos` 

I guess you testing is failing due to `opam install .` expecting the sources of Coq to be there too.

Reply:
Ok! I can now properly test things, thank you very much.

As you predicted, my "fix" doesn't fix anything. The env log shows `COQ_USE_DUNE` set to false, but `dune` is clearly being used anyway.

Reply:
Yup, you need to do `unset COQ_USE_DUNE && make` in the build field.

Reply:
Umm, no idea; does `make "COQ_USE_DUNE=" "-j%{jobs"` work locally? If so, that should may be the preferred form.

Reply:
It... almost works? Your example made me realize that `[make "-j%{jobs}%" "COQ_USE_DUNE=false"]` works as well, I was just too pessimistic to actually read the error message and realize it was a new, different error. I'm now stuck at `make install-byte`, for some as yet unknown reason.

Reply:
You need to add that line to all calls to make unfortunately, but now that I think, maybe setting in the `env` part as you did first `[ COQ_USE_DUNE= ]` would actually work too.

Reply:
Yes, adding it to the `make install byte` line made everything actually work! For some reason it wasn't needed in the `make install` line (???).

> maybe setting in the env part as you did first [ COQ_USE_DUNE= ] would actually work too.

I will try it for curiosity's sake, but is that really different from `[COQ_USE_DUNE=false]`?

Reply:
Yes, because the makefile has `ifdef COQ_USE_DUNE`

Reply:
By the way I'm sure it is needed in the `install` call too, if you that route; likely omitting it will cause the install target to turn into a no-op

Reply:
> maybe setting in the env part as you did first [ COQ_USE_DUNE= ] would actually work too.

It seems that `opam` does not allow this; it gives a syntax error, even with `[ COQ_USE_DUNE="" ]`. So I guess we're going with the other option.

Reply:
Pity, that would have been clearer. So indeed we need undefine the var in every call to make; annoying ;S